ecs-codedeploy

```diff
@@ -38,7 +38,7 @@ function print_usage {
   echo
   echo "Examples:"
   echo
-  echo "  ecs-codedeploy --iam-role arn:aws:iam::604506250246:role/ecs-deploy --cluster production --region us-west-2 --service backend --image 640673857988.dkr.ecr.eu-central-1.amazonaws.com/backend:v1.2.3"
```

Comment:
I'm still pretty foreign to this code deploy stuff, but do we no longer need an iam role?

Reply:
We don't, this was an option I toyed with, but i'm not implementing it right now.